build.gradle

```diff
@@ -19,17 +19,6 @@ repositories {
 	}
 }
 
-// overwrite jastadd2 dependency set by jastaddgradle
-configurations {
-	all {
-		resolutionStrategy.eachDependency { DependencyResolveDetails details ->
```

Comment:
What happened with this part? Is this not needed any more due to updates in jastaddgradle? Please include the rationale for this in the commit comment.